src/libstd/process.rs

```diff
@@ -400,6 +401,33 @@ pub struct Output {
     pub stderr: Vec<u8>,
 }
 
+//If either stderr or stdout are valid utf8 strings it prints the valid strings, otherwise it
+//prints the byte sequence instead
```

Comment:
Stylistically there's typically a space after the `//`, and can you also format this comment to a maximum width of 80 chars? (the surrounding style)